Documentation/project-docs/contributing.md

```diff
@@ -131,6 +109,28 @@ The following file header is the used for .NET Core. Please use it for new files
 - See [class.cpp](../../src/vm/class.cpp) for an example of the header in a C++ file.
 - See [List.cs](../../src/mscorlib/src/System/Collections/Generic/List.cs) for an example of the header in a C# file.
 
+Contributing Ports
+------------------
+
+We encourage ports of CoreCLR to other platforms. There are multiple ports ongoing at any one time. You may be interested in one of the following ports:
+
+Chips:
+
+- [ARM32](https://github.com/dotnet/coreclr/labels/arch-arm32)
+- [ARM64](https://github.com/dotnet/coreclr/labels/arch-arm64)
+- [X86](https://github.com/dotnet/coreclr/labels/arch-x86)
+
+Operating System:
+
+- [Linux](https://github.com/dotnet/coreclr/labels/os-linux)
```

Comment:
Bit unclear what Linux and macOS are doing here. Maybe Linux distros, but that label is really just used for Linux specific bugs. Maybe list potential distros? And is there any porting relevant to macOS?